integrations-and-supported-tools/plotly/scripts/requirements.txt

```diff
@@ -1,2 +1,2 @@
-neptune-client==0.9.5
-plotly==4.4.1
\ No newline at end of file
+neptune-client
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works.
We'll be updating versions periodically.